src/OpenDebugAD7/AD7DebugSession.cs

```diff
@@ -2076,6 +2076,16 @@ protected override void HandleDisassembleRequestAsync(IRequestResponder<Disassem
                         Instruction = data.bstrOpcode,
                         Symbol = data.bstrSymbol
                     };
+
+                    if (!string.IsNullOrWhiteSpace(data.bstrDocumentUrl))
+                    {
+                        instruction.Location = new Source()
+                        {
+                            Path = data.bstrDocumentUrl
+                        };
+                        instruction.Line = (int)data.posBeg.dwLine;
```

Comment:
>(int)data.posBeg.dwLine [](http://example.com/codeflow?start=43&length=23)

m_pathConverter.ConvertDebuggerLineToClient

Reply:
Which might mean that you are missing a `- 1` in the line calculations in AD7Disassembly.